### PR TITLE
Update field-selectors.md

### PR DESCRIPTION
**Pull request by liuhanguang123**
Update the bracket on the right to become tilted

### CODE REVIEW

**Comment by liuhanguang123** [comment]
@mengjiao-liu

File: content/zh/docs/concepts/overview/working-with-objects/field-selectors.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ weight: 60
 <!--
 _Field selectors_ let you [select Kubernetes resources](/docs/concepts/overview/working-with-objects/kubernetes-objects) based on the value of one or more resource fields. Here are some example field selector queries:
 -->
-_字段选择器（Field selectors_）允许你根据一个或多个资源字段的值
+_字段选择器（Field selectors）_ 允许你根据一个或多个资源字段的值
```
Comment (tengqm) [inline content/zh/docs/concepts/overview/working-with-objects/field-selectors.md:13]:
```suggestion
“字段选择器（Field selectors）”允许你根据一个或多个资源字段的值
```

**Comment by mengjiao-liu** [comment]
/approve
/lgtm
Thanks!

**Pull request merged**